screens/login.tsx

```diff
@@ -1,20 +1,97 @@
 import { StatusBar } from "expo-status-bar";
 import React from "react";
-import { StyleSheet, Text, View } from "react-native";
+import { StyleSheet, Text, View, Image, TextInput, Button } from "react-native";
+import { Formik } from "formik";
+import LoginButton from "../components/button";
+import PasswordButton from "../components/password_button";
+import SignUpButton from "../components/sign_up_button";
 
 export default function Login() {
   return (
     <View style={styles.container}>
-        // TODO
+      <View>
+        <Text style={styles.heading}>Welcome Back!</Text>
+        <Image
+          style={styles.image}
+          source={require("../assets/undraw_breakfast.png")}
+        />
+        <Formik
+          initialValues={{ email: "", password: "" }}
+          onSubmit={(values) => {}}
+        >
+          {(props) => (
+            <View>
+              <TextInput
+                style={styles.input}
+                placeholder="Enter your school email"
+                onChangeText={props.handleChange("email")}
+                value={props.values.email}
+              />
+              <TextInput
+                style={styles.input}
+                placeholder="Enter password"
+                onChangeText={props.handleChange("password")}
+                value={props.values.password}
+              />
+              <PasswordButton onPress={() => {}} />
+              <LoginButton text="Sign In" onPress={() => {}} />
```

Comment:
I'd say maybe move things down (look at the Figma) the button and above are a bit lower.